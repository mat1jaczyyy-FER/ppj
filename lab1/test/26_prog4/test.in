a = 12
b = a + 12
c = b*13
d = a/b/c + d
do = od
od = 3+zaz
