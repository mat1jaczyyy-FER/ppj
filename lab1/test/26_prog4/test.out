IDN 1 a
OP_PRIDRUZI 1 =
BROJ 1 12
IDN 2 b
OP_PRIDRUZI 2 =
IDN 2 a
OP_PLUS 2 +
BROJ 2 12
IDN 3 c
OP_PRIDRUZI 3 =
IDN 3 b
OP_PUTA 3 *
BROJ 3 13
IDN 4 d
OP_PRIDRUZI 4 =
IDN 4 a
OP_DIJELI 4 /
IDN 4 b
OP_DIJELI 4 /
IDN 4 c
OP_PLUS 4 +
IDN 4 d
KR_DO 5 do
OP_PRIDRUZI 5 =
KR_OD 5 od
KR_OD 6 od
OP_PRIDRUZI 6 =
BROJ 6 3
OP_PLUS 6 +
IDN 6 zaz
